android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/components/MeasureReportIndividualResultView.kt

```diff
@@ -54,11 +54,11 @@ const val RESULT_VIEW_INDICATOR_DESCRIPTION = "resultViewIndicatorDescription"
 
 @Composable
 fun MeasureReportIndividualResultView(
-  modifier: Modifier = Modifier,
-  patientViewData: MeasureReportPatientViewData,
-  isMatchedIndicator: Boolean = true,
-  indicatorStatus: String = "",
-  indicatorDescription: String = ""
+    modifier: Modifier = Modifier,
+    patientViewData: MeasureReportSubjectViewData,
```

Comment:
should we update the key too?
```suggestion
    subjectViewData: MeasureReportSubjectViewData,
```